### ISSUE TITLE
Errors on payload when using a `working_directory` in Circle 2

### ISSUE THREAD

**Issue by jhanggi** [rdunlop/codeclimate_circle_ci_coverage#8]
If you set a `working_directory` in your Circle config, then `codeclimate_circle_ci_coverage` is looking in the wrong place for the artifacts and we are seeing this:

```
bundler: failed to load command: report_coverage (/home/circleci/nucore/vendor/bundle/ruby/2.4.0/bin/report_coverage)
CodeClimate::TestReporter::InvalidPayload: No source files were found in the test report payload
```

See https://circleci.com/gh/tablexi/nucore-open/6416#artifacts/containers/0 which has a `working_directory: ~/nucore`. The artifacts are then placed in `home/circleci/nucore/coverage/.resultset.json`

**Comment by tjwp** [comment]
I ran into this issue and was able to workaround it with the following configuration:

```
- store_artifacts:
    path: /path/to/working/directory/coverage/.resultset.json
    destination: home/circleci/project/coverage/.resultset.json
```

**Comment by rdunlop** [comment]
resolved by #9
